src/c++/perf_analyzer/client_backend/openai/http_client.cc

```diff
@@ -0,0 +1,267 @@
+// Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Why not include this client with the other C++ clients in `src/c++/library`?

Reply:
Only the triton-specific client is there. All other clients are in client_backend/*/. This is something that needs to be cleaned up and refactored later